formfyxer/data/gendered_terms.yml

```diff
@@ -0,0 +1,318 @@
+sisterhood: siblinghood
+man (verb): to guard
+woman-made: synthetic
+womanpower: personpower
+meter-maid: meter-reader
+motherland: homeland
+fatherland: homeland
+alderwoman: alderperson
+bondswoman: bondsperson
+boss woman: boss
+brakewoman: brake operator
+chairwoman: chair
+councilman: councilperson
+countryman: countryperson
+dairywoman: dairyperson
+everywoman: everyperson
+freshwoman: first-year student
+handywoman: handyperson
+horsewoman: horseperson
+lineswoman: lineperson
+presswoman: press operator
+schoolgirl: schoolchild
+watchwoman: watchperson
+weatherman: weatherperson
+chorusgirl: chorusperson
+boy Friday: aide
+officegirl: officeperson
+comedienne: comedian
+sculptress: sculptor
+stewardess: steward
+brewmaster: brewmaker
+headmaster: headmaker
+shipmaster: shipmaker
+taskmaster: taskmaker
+man to man: person to person
+bridesmaid: attendant
+flower boy: flower child
+stag party: animal party
+sex change: gender affirmation
+preference: orientation
+boyfriend: friend
+womankind: people
+brotherly: siblingly
+meter-man: meter-reader
+anchorman: anchor
+cameraman: cameraperson
+careerman: careerperson
+cavewoman: cave person
+clergyman: clergyperson
+craftsman: craftsperson
+crewwoman: crewperson
+doorwoman: doorperson
+draftsman: draftsperson
+firewoman: firefighter
+fisherman: fisherperson
+forewoman: foreperson
+gentleman: gentleperson
+kinswoman: kinsperson
+letterman: letterperson
+linewoman: lineperson
+mailwoman: mail carrier
+middleman: intermediary
+newswoman: newsperson
+ombudsman: ombudsperson
+patrolman: patrolperson
+policeman: police officer
+postwoman: mail carrier
+repairman: repairperson
+schoolboy: schoolchild
+snowwoman: snowperson
+spokesman: spokesperson
+sportsman: sportsperson
+statesman: statesperson
+cabingirl: cabinperson
+choirgirl: choirperson
+chorusboy: chorusperson
+officeboy: officeperson
+papergirl: paperperson
+conductor: conductor
+majorette: major
+patroness: patron
+priestess: priest
+sorceress: sorcerer
+temptress: tempter
+usherette: usher
+best maid: best person
+groomsman: attendant
+groomsmen: attendants
+hen party: animal party
+lifestyle: orientation
+daughter: child
+brethren: laity
+sisterly: siblingly
+man-made: synthetic
+manpower: personpower
+ad-woman: ad-person
+airwoman: airperson
+alderman: alderperson
+barwoman: barperson
+bondsman: bondsperson
+boss man: boss
+brakeman: brake operator
+chairman: chair
+dairyman: dairyperson
+everyman: everyperson
+fall guy: fall person
+fall gal: fall person
+freshman: first-year student
+handyman: handyperson
+horseman: horseperson
+laywoman: layperson
+lawwoman: lawperson
+linesman: lineperson
+pressman: press operator
+watchman: watchperson
+bellgirl: bellperson
+cabinboy: cabinperson
+choirboy: choirperson
+copygirl: copyperson
+newsgirl: newsperson
+paperboy: paperperson
+playgirl: playperson
+comedian: comedian
+masseuse: masseur
+sculptor: sculptor
+sorcerer: sorcerer
+shepherd: shepherd
+waitress: waiter
+to a man: to a one
```

Comment:
I think I would find this confusing, though I suspect it's unlikely to turn up in legal forms anyway? Could this just be an empty string? It's redundant, right, and used mostly as emphasis? If needed, I think something like "every one of them" could be less confusing.

Reply:
The list of gendered terms should be considered separately from the list of "complex" words with simple alternatives. I tried not to apply too much of my own judgment